src/main/java/in/projecteka/consentmanager/user/SignUpService.java

```diff
@@ -22,21 +23,14 @@
 import java.util.concurrent.ExecutionException;
 import java.util.function.Function;
 
+@AllArgsConstructor
 public class SignUpService {
 
     private final static Logger logger = Logger.getLogger(SignUpService.class);
+    public static final long JWT_TOKEN_VALIDITY = 5 * 60 * 60;
```

Comment:
if its duration, then this should not be hard coded. Should be loaded from application property.

Reply:
Done.